### PR TITLE
clean up trailing '/' in path before routing

### PR DESCRIPTION
**Pull request by kickster97**
### WHAT is this pull request doing?
clean up any trailing '/' in path before routing, fixes failing rabbit-hole specs

### HOW can this pull request be tested?
run rabbit-hole specs

### CODE REVIEW

File: src/lavinmq/http/router.cr [hunk 1]
```diff
@@ -33,7 +33,7 @@ module LavinMQ::HTTP::Router
   {% end %}
 
   def find_route(method, path)
-    search_path = "/#{method}#{path}"
+    search_path = "/#{method}#{path.sub(/\/$/, "")}"
```
Comment (spuun) [inline src/lavinmq/http/router.cr:36]: There is a method for this :)
```suggestion
    search_path = "/#{method}#{path.rstrip('/')}"
```

**Review by spuun — COMMENTED** [review]
It's even simpler!

**Comment by spuun** [comment]
Oh, actually, we should only rstrip if it ends with slash!

**Comment by spuun** [comment]
> Oh, actually, we should only rstrip if it ends with slash!

Uhm, and if path is longer than 1. Maybe just do 
```
    search_path = "/#{method}/#{path.strip('/')}"
```

**Comment by kickster97** [comment]
> search_path = "/#{method}/#{path.strip('/')}"

ah, assuming this is for handling the `get "/"` cases?

**Comment by spuun** [comment]
> > search_path = "/#{method}/#{path.strip('/')}"
> 
> ah, assuming this is for handling the `get "/"` cases?

Yes!

**Review by spuun — APPROVED** [review]
Let's add a spec for the "/" case, then merge!

**Review by spuun — APPROVED** [review]

**Pull request merged**